stdlib/public/core/Availability.swift

```diff
@@ -17,7 +17,9 @@ import SwiftShims
 ///
 /// This is a magic entry point known to the compiler. It is called in
 /// generated code for API availability checking.
-@inlinable // FIXME(sil-serialize-all)
+/// Note: It is important not to make this function inlinable. There is a pass
+/// that relies on being able to tell whether this function is called. It does
+/// this using the semantics attribute.
```

Comment:
Then you'll have to make it `inline(never)`, or else it could still be inlined within the standard library.

Reply:
Good point. For now we don't use #available in the standard library. But we could at a later point. I will fix in a follow up.

Reply:
#17877